index.html

```diff
@@ -823,46 +825,43 @@ <h2>Identifiers</h2>
         </pre>
 
         <p>
-The example above uses two types of identifiers. The first identifier is an
-identifier for the credential and uses an HTTP-based URL. The second
-identifier is an identifier for the subject of the credential, that is, the
-thing that the claims are about, which uses a Decentralized Identifier, also
-known as a DID.
+The example above uses two types of identifiers. The first identifier is for 
+the <a>credential</a> and uses an HTTP-based URL. The second identifier is for 
+the <a>subject</a> of the <a>credential</a> (the thing the <a>claims</a> are 
+about) and uses a Decentralized Identifier, also known as a DID.
         </p>
 
         <p class="note">
-Note that, as of this publication, DIDs are a new type
-of identifier that are not necessary for Verifiable Credentials to be useful.
-For the avoidance of doubt, Verifiable Credentials do not depend on
-Decentralized Identifiers and Decentralized Identifiers do not depend on
-Verifiable Credentials.
-However, it is expected that many Verifiable Credentials will use
-Decentralized Identifiers and software libraries that implement this
-specification will most likely need to be able to resolve DIDs. DID-based
+As of this publication, DIDs are a new type of identifier that are not 
+necessary for <a>verifiable credentials</a> to be useful. Specifically, 
+<a>verifiable credentials</a> do not depend on DIDs and DIDs do not depend on 
+<a>verifiable credentials</a>. However, it is expected that many 
+<a>verifiable credentials</a> will use DIDs and software libraries 
+implementing this specification will probably need to resolve DIDs. DID-based
 URLs are used for expressing identifiers associated with cryptographic keys
 and other machine-readable information that is useful when used in
-the Verifiable Credentials Ecosystem.
+the <a>verifiable credentials</a> ecosystem.
         </p>
       </section>
 
       <section>
         <h2>Types</h2>
 
         <p>
-Software systems that process the objects specified in this document
-use type information to make determinations about whether or not the provided
-<a>credential</a> or <a>presentation</a> is appropriate.
-Type information MUST be expressed via the <a>type</a> property:
+Software systems that process the kinds of objects specified in this document
+use type information to determine whether or not a provided <a>credential</a> 
+or <a>presentation</a> is appropriate. Type information MUST be expressed 
+using the <code>type</code> property.
         </p>
 
         <dl>
           <dt><dfn>type</dfn></dt>
```

Comment:
```suggestion
           <dt><dfn data-lt="type|types">type</dfn></dt>
```
How about adding the plural here so `<a>types</a>` can be used vs `<a>type</a>s`.  If this is added could fix the various places it's used.